src/stp/new_plays/Halt.cpp

```diff
@@ -0,0 +1,66 @@
+//
+// Created by jessevw on 24.03.20.
+//
+
+#include "include/roboteam_ai/stp/new_plays/Halt.h"
+#include <stp/new_roles/TestRole.h>
+#include "stp/new_roles/Halt.h"
+namespace rtt::ai::stp::play {
+
+    Halt::Halt() {
+        roles = std::array<std::unique_ptr<Role>, rtt::ai::Constants::ROBOT_COUNT()>{
+                std::make_unique<role::Halt>(role::Halt("halt0")), std::make_unique<role::Halt>(role::Halt("halt1")),
```

Comment:
are we doing role_name_0 or roleName0? I don't mind either but just for consistency

Reply:
Nice catch! Updated this as well :)